src/agni/server/receiver/HeartbeatReceiver.java

```diff
@@ -4,20 +4,20 @@
 
 public class HeartbeatReceiver implements MessageParser {
 
-    private Vector <StatusListener> statusListeners = null;
+    private Vector <HeartbeatListener> heartbeatListeners = null;
 
     public HeartbeatReceiver() {
-        statusListeners = new Vector<StatusListener>();
+        heartbeatListeners = new Vector<HeartbeatListener>();
     }
 
     private void notifyHeartbeat(String ip, byte status) {
```

Comment:
Since the HeartbeatManager is different from the StatusManager, should we remove this argument for the listener? It was originally used to give information about whether a user was offline, away, or online.